dashboards-observability/public/components/visualizations/charts/financial/candle_stick/candle_stick.tsx

```diff
@@ -0,0 +1,217 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+
+import { Plt } from '../../../plotly/plot';
+
+export const CandleStick = ({ visualizations, layout, config }) => {
```

Comment:
what is this?

Reply:
oh this was added previously but now will be removed from 1.3 release, i'll remove the entire visualization